### ISSUE TITLE
mIoU is only 0.579 much lower than your reported 0.69

### ISSUE THREAD

**Issue by l44xu** [meng-tang/rloss#13]
Hi Meng,

I tried the pytorch code  using exactly the instructed steps. With 60 epochs, I only got mIOU of 0.579, much lower than your reported 0.69. Is there something that I should do to improve the accuracy?

I used the below command:

python train_withdensecrfloss.py --backbone mobilenet --lr 0.007 --workers 6 --epochs 60 --batch-size 8  --checkname deeplab-mobilenet --eval-interval 2 --dataset pascal --save-interval 2 --densecrfloss 2e-9 --rloss-scale 0.5 --sigma-rgb 15 --sigma-xy 100

Many thanks!

Best,
Linlin

**Comment by meng-tang** [comment]
For debugging, could you try to train without any regularization, just the partial cross-entropy loss? I would expect 65.x% in this case.
